src/features/projectsV2/ProjectDetails/components/OtherInterventionInfo.tsx

```
@@ -110,19 +110,19 @@ const OtherInterventionInfo = ({
   selectedPlantLocation,
-  hoveredPlantLocation
+  hoveredPlantLocation,
 }: Props) => {
   const plantLocationInfo = hoveredPlantLocation || selectedPlantLocation;
   const tProjectDetails = useTranslations('ProjectDetails');
   const { totalTreesCount } = useMemo(() => {
     const totalTreesCount =
       plantLocationInfo &&
-        plantLocationInfo.plantedSpecies &&
-        plantLocationInfo.plantedSpecies.length > 0
+      plantLocationInfo.plantedSpecies &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)